src/Appwrite/Extend/Exception.php

```diff
@@ -273,4 +278,14 @@ public function setType(string $type): void
     {
         $this->type = $type;
     }
+
+    /**
+     * Get the log of the exception.
+     *
+     * @return bool
+     */
+    public function getLog(): bool
```

Comment:
This is a misleading name for the getter.

The `log` attribute is a boolean that controls whether a log should be pushed or not... But the name `log` does not indicate this...  we've to use a better name for the attribute and thereby the getter

Reply:
Changing it to `publishLog`